packages/ember-test-app/tests/integration/components/card-test.ts

```diff
@@ -0,0 +1,27 @@
+import { assert, module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | components | card', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders the card with the correct content', async function () {
+    await render(hbs`<Card>
+      <:card-header>
+        <p>Header content</p>
+      </:card-header>
+      <:card-body>
+        <p>Body content</p>
+      </:card-body>
+    </Card>`);
```

Comment:
Let's format these a little more. Here's an example:
```suggestion
    await render(hbs`
      <Card>
        <:card-header>
          <p>Header content</p>
        </:card-header>
        <:card-body>
          <p>Body content</p>
        </:card-body>
      </Card>
    `);
```

Reply:
Fixed